pkg/endpoint/endpoint.go

```diff
@@ -476,6 +484,48 @@ type Endpoint struct {
 	DeprecatedOpts deprecatedOptions `json:"Opts"`
 }
 
+// GetIngressPolicyEnabledUnlocked returns whether ingress policy
+// enforcement is enabled for endpoint or not. The endpoint's mutex must be
+// held.
+func (e *Endpoint) GetIngressPolicyEnabledUnlocked() bool {
```

Comment:
Is this supposed to be `GetIngressPolicyEnabledLocked()`?

Reply:
You’re right, I was using Unlocked to refer to no locking in the function itself, but that’s inconsistent with the rest of the code and is confusing. I’ll change it.